cohere/client.py

```diff
@@ -153,7 +153,10 @@ def chat(self,
              persona: str = "cohere",
              model: Optional[str] = None,
              return_chatlog: bool = False,
-             chatlog_override: Optional[List[Dict[str, str]]] = None) -> Chat:
+             return_prompt: bool = False,
```

Comment:
why is this a parameter at all?

Reply:
Prompt construction is opaque in the chat endpoint since what you pass in to the endpoint is not equal to the full prompt. 

So this is primarily to help devs debug -- we had it requested by a few users 😁.

Reply:
just reflecting the endpoint updates, this is mainly for debugging purposes. [here](https://linear.app/cohereai/issue/CNV-287/add-return-prompt-to-the-python-sdk#comment-c9d4b7ce)'s a brief discussion about it